src/java/org/apache/cassandra/config/GuardrailsOptions.java

```diff
@@ -1164,8 +1164,8 @@ private static void validateWarnLowerThanFail(long warn, long fail, String name)
             return;
 
         if (fail < warn)
-            throw new IllegalArgumentException(format("The warn threshold %d for %s_warn_threshold should not be " +
-                                                      "higher than the fail threshold %d", warn, name, fail));
+            throw new IllegalArgumentException(format("The warn threshold %d for %s_warn_threshold should be lower " +
+                                                      "than the fail threshold %d", warn, name, fail));
```

Comment:
@adelapena @bereng @smiklosovic I don't really want to fix it in this patch, but the check and the warning message don't strictly agree. Either it should be `if (fail <= warn)` or the error message needs to change.

Reply:
If you find a bug you fix it... I don't see a problem with that.